src/base/_type.css

```diff
@@ -0,0 +1,127 @@
+/*
+
+  TYPE
+
+*/
+
+:root {
+  --font-weight-body: 200;
+  --font-weight-headings: 200;
+  --font-family-monospace: Menlo, Consolas, Monaco, monospace;
+}
+
+html {
+  line-height: 1.5; /* .lh-copy */
```

Comment:
This would be preferable. However, tachyons doesn't have variables for these already, so it'd still be looking at what is set for tachyons and setting them as the variables, as opposed to just setting them on the element.

They do for margins and things, just not for everything (which is honestly, kind of annoying)